templates/shortcodes/todo.html

```diff
@@ -0,0 +1,3 @@
+<div class="todo">
+	<h2 class="todo-header">TODO</h2>{{ body | markdown | safe }}
```

Comment:
```suggestion
    <h2 class="todo-header">TODO</h2>
    {{ body | markdown | safe }}
```
Spaces and newline for content